### ISSUE TITLE
Truncated Prism `shell-session` language command output

### ISSUE THREAD

**Issue by sunknudsen** [react-syntax-highlighter/react-syntax-highlighter#287]
**Describe the bug**

When using Prism `shell-session` language, command output is truncated.

**To Reproduce**

Supply the following value to `<SyntaxHighlighter language="shell-session">{value}</SyntaxHighlighter>`.

```
$ cd ~/.ssh
$ ssh-keygen -t rsa -C "vpn-server"
Generating public/private rsa key pair.
Enter file in which to save the key (/Users/sunknudsen/.ssh/id_rsa): vpn-server
Enter passphrase (empty for no passphrase):
Enter same passphrase again:
Your identification has been saved in vpn-server.
Your public key has been saved in vpn-server.pub.
The key fingerprint is:
SHA256:4On7WymZIcM5p8SbsybwJpaFIUrnTUMf/1fdAhI1WPY vpn-server
The key's randomart image is:
+---[RSA 3072]----+
|          .==    |
|     . .  o..o   |
|    . o o  . .E o|
|.... * = .    ..o|
|o.ooo % S .   .. |
|. o..+ O + o .   |
|   =  * + o .    |
|  + + .+ o       |
| . o oo.o.       |
+----[SHA256]-----+
```

**Expected behavior**

The `<span class="token output">` tag should include the whole output of command `$ ssh-keygen -t rsa -C "vpn-server"` as seen on https://prismjs.com/test.html#language=shell-session (see screenshot).

Expected HTML:

```html
<pre class="language-shell-session"><code class="  language-shell-session"><span class="token command"><span class="token shell-symbol important">$</span> <span class="token bash language-bash"><span class="token builtin class-name">cd</span> ~/.ssh</span></span>
<span class="token command"><span class="token shell-symbol important">$</span> <span class="token bash language-bash">ssh-keygen -t rsa -C <span class="token string">"vpn-server"</span></span></span>
<span class="token output">Generating public/private rsa key pair.
Enter file in which to save the key (/Users/sunknudsen/.ssh/id_rsa): vpn-server
Enter passphrase (empty for no passphrase):
Enter same passphrase again:
Your identification has been saved in vpn-server.
Your public key has been saved in vpn-server.pub.
The key fingerprint is:
SHA256:4On7WymZIcM5p8SbsybwJpaFIUrnTUMf/1fdAhI1WPY vpn-server
The key's randomart image is:
+---[RSA 3072]----+
|          .==    |
|     . .  o..o   |
|    . o o  . .E o|
|.... * = .    ..o|
|o.ooo % S .   .. |
|. o..+ O + o .   |
|   =  * + o .    |
|  + + .+ o       |
| . o oo.o.       |
+----[SHA256]-----+</span></code></pre>
```

Instead, the `<span class="token output">` tag only includes `Generating public/private rsa key pair.`.

```html
<pre style="background-color: rgb(255, 255, 255);"><code><span class="token command token shell-symbol important">$</span><span class="token command"> </span><span class="token command token bash language-bash token builtin class-name">cd</span><span class="token command token bash language-bash"> ~/.ssh</span><span>
</span><span></span><span class="token command token shell-symbol important">$</span><span class="token command"> </span><span class="token command token bash language-bash">ssh-keygen -t rsa -C </span><span class="token command token bash language-bash token string">"vpn-server"</span><span>
</span><span></span><span class="token output">Generating public/private rsa key pair.
</span>Enter file in which to save the key (/Users/sunknudsen/.ssh/id_rsa): vpn-server
Enter passphrase (empty for no passphrase):
Enter same passphrase again:
Your identification has been saved in vpn-server.
Your public key has been saved in vpn-server.pub.
The key fingerprint is:
SHA256:4On7WymZIcM5p8SbsybwJpaFIUrnTUMf/1fdAhI1WPY vpn-server
The key's randomart image is:
+---[RSA 3072]----+
|          .==    |
|     . .  o..o   |
|    . o o  . .E o|
|.... * = .    ..o|
|o.ooo % S .   .. |
|. o..+ O + o .   |
|   =  * + o .    |
|  + + .+ o       |
| . o oo.o.       |
+----[SHA256]-----+
</code></pre>
```

**Screenshots**

![prismjs](https://user-images.githubusercontent.com/2117655/89358807-d50b5080-d6b3-11ea-9d3f-243e728fbde8.png)

**Desktop (please complete the following information):**

 - OS: macOS 10.15.6
 - Browser: Firefox
 - Version: 79.0 (64-bit)

**Comment by sunknudsen** [comment]
FYI: using `react-refractor` instead of `react-syntax-highlighter` works so I believe the issue is within this package.

```tsx
import Refractor from "react-refractor"
import shellSession from "refractor/lang/shell-session"

Refractor.registerLanguage(shellSession)

<Refractor
  language="shell-session"
  value={value}
 />
```

**Comment by simmerer** [comment]
Thanks for doublechecking against `react-refractor`, @sunknudsen 

I'll try to dig into this soon.

**Comment by simmerer** [comment]
Great news: @urish 's fix over here https://github.com/react-syntax-highlighter/react-syntax-highlighter/pull/289 also solved the root cause of this issue!

I'll get these rolled into a release shortly.

**Comment by simmerer** [comment]
Fixed in 13.3.1
